### PR TITLE
Fixed some doc issues

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: Resources/doc/groups.rst [hunk 1]
```diff
@@ -22,25 +22,25 @@ Below is an example configuration for enabling groups support.
...
-        <fos_user:group group-class="AppBundle\Entity\Group" />
-    </fos_user:config>
+        <!-- app/config/config.xml -->
+        <fos_user:config
+            db-driver="orm"
+            firewall-name="main"
+            user-class="AppBundle\Entity\User"
+        >
+            <fos_user:group group-class="AppBundle\Entity\Group" />
+        </fos_user:config>
```
Comment (xabbuh) [inline Resources/doc/groups.rst]: What about providing a complete (i.e. ready for copy and paste) example here?

File: Resources/doc/overriding_controllers.rst [hunk 2]
```diff
@@ -28,7 +28,7 @@ bundle named ``AcmeUserBundle`` that declares itself a child of FOSUserBundle.
 
 .. note::
 
-    The Symfony2 framework only allows a bundle to have one child. You cannot
+    The Symfony framework only allows a bundle to have one child. You cannot
```
Comment (xabbuh) [inline Resources/doc/overriding_controllers.rst]: Symfony Framework (same in other places)

**Comment by stof** [comment]
@javiereguiluz cn you fix the conflicts ?

**Comment by stof** [comment]
@javiereguiluz ping

**Comment by stof** [comment]
@javiereguiluz ping again

File: Resources/doc/groups.rst (hunk 1 above)
Comment (javiereguiluz) [inline Resources/doc/groups.rst]: This example is not ready to copy/paste?

**Comment by javiereguiluz** [comment]
Rebased and fixed some issues. There is still a pending comment from @xabbuh which I don't understand.

**Comment by stof** [comment]
I will merge it as is. Once @xabbuh gives more details, a separate PR could be done to fix this.

**Comment by stof** [comment]
Thank you @javiereguiluz.

**Comment by xabbuh** [comment]
see #1927

**Pull request closed without merging**